test/turnt.toml

```diff
@@ -81,14 +81,13 @@ command = "python3 ../slow_odgi/overlap.py {base}.paths < {filename}"
 
 [envs.validate_setup]
 binary = true
-command = "python3 ../slow_odgi/perturb.py < {filename} > temp.{filename}"
+command = "python3 ../slow_odgi/perturb.py < {filename}"
+output.gfa = "-"
```

Comment:
A silly workaround, but I use the output.extension trick to clobber the GFAs in place. This means that the GFAs need to be re-fetched for future commands, but it does help avoid the situation where we were creating new temp gfas and then seeing turnt use those temp gfas as targets.

Reply:
Something like this makes sense to me! Maybe deserves its own PR though?

Reply:
Gotcha, will do!